payjoin/src/lib.rs

```diff
@@ -5,17 +5,14 @@
 //! While I don't think there's a huge risk running it, don't rely on its security for now!
 //! Please at least review the code that verifies there's no overpayment and let me know you did.
 //!
-//! This is a library and an example binary implementing BIP78 Payjoin.
+//! This is a library and an example binary implementing BIP78 and BIP77 Payjoin.
+//!
 //! The library is perfectly IO-agnostic - in fact, it does no IO.
 //! The primary goal of such design is to make it easy to unit test.
 //! While we're not there yet, it already has infinitely more tests than the Payjoin PR against Electrum. :P
 //!
 //! Additional advantage is it doesn't care whether you use `async`, blocking, `tokio`, `sync-std` `hyper`, `actix` or whatever.
 //! There are already too many frameworks in Rust so it's best avoiding directly introducing them into library code.
-//!
```

Comment:
Generally in order to see a crate features one would look into the `cargo.toml` or in the `features` section in `docs.rs`. If we still want to mention features in docs, I would suggest to mention all of them, otherwise it will be misleading and could lead to users thinking no other features exist than the ones in the docs

Reply:
![image](https://github.com/payjoin/rust-payjoin/assets/8525467/2601c0f4-a4c6-451d-9288-0355ab24ac24)
No, generally features are at least outlined in docs.rs. If we want to mention `v2` or `base64` that is separate from this pr